.github/workflows/test-pydantic1.yaml

```diff
@@ -51,7 +51,7 @@ jobs:
       # install pydantic
       #----------------------------------------------
       - name: Install Pydantic
-        run: poetry add pydantic@^1
+        run: poetry run pip install 'pydantic<2'
```

Comment:
Just curious - why use pip to install pydantic into the poetry env instead of poetry?

Reply:
Bc the add fails here since numpydantic requires pydantic 2, the pip install just updates pydantic